parm/config/gefs/config.efcs

```diff
@@ -66,32 +66,13 @@ export SPPT_LSCALE=500000.
 export SPPT_LOGIT=".true."
 export SPPT_SFCLIMIT=".true."
 
-if [ $QUILTING = ".true." -a $OUTPUT_GRID = "gaussian_grid" ]; then
-    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_da"
+if [[ ${QUILTING} = ".true." && ${OUTPUT_GRID} = "gaussian_grid" ]]; then
+    export DIAG_TABLE="${HOMEgfs}/parm/parm_fv3diag/diag_table_da"
 else
-    export DIAG_TABLE="$HOMEgfs/parm/parm_fv3diag/diag_table_da_orig"
+    export DIAG_TABLE="${HOMEgfs}/parm/parm_fv3diag/diag_table_da_orig"
```

Comment:
Dial this down to one table.  There is no `_da` counterpart in GEFS

Reply:
And done